sonar-project.properties

```diff
@@ -0,0 +1,7 @@
+sonar.projectKey=open-cluster-management_search-e2e-test
```

Comment:
`sonar.projectKey=stolostron_search-e2e-test`
```suggestion
sonar.projectKey=stolostron_search-e2e-test
```

Reply:
Some of our new projects, including search-v2-api, had stolostron_. But e2e test seems to be open-cluster-management_*
<img width="296" alt="image" src="https://user-images.githubusercontent.com/32376120/151995329-e2adb2da-e265-49ca-8f1a-23ee8f3ca4b0.png">